gitlab/tests/mixins/test_object_mixins_attributes.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
```

Comment:
Where does this license header come from?

Reply:
Oops I must have left this when splitting the files from the original mixin tests. I've removed it now but I can also add new headers to all the new files if needed.